CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGE LOG
 
+### v0.4.17
```

Comment:
I think the commit is not up to date, if you merge master to SEB-1379-crowdstrike and push again it should fix the issue.

Reply:
Bumped version up to 0.4.19?

Reply:
yes this is correct!